app/javascript/components/patient/Patient.js

```diff
@@ -671,7 +671,7 @@ class Patient extends React.Component {
 
 Patient.propTypes = {
   details: PropTypes.object,
-  jurisdiction_path: PropTypes.string,
+  jurisdiction_paths: PropTypes.object,
```

Comment:
the household member table needs all the paths to check if each household member is in a viewable jurisdiction for the current user.  this change will also be needed for @hackrm's follow up flag PR

Reply:
You make a point, I guess it really wouldn't be needed [here](https://github.com/SaraAlert/SaraAlert/blob/844-add-household-members-to-dependent/app/javascript/components/patient/household/utils/HouseholdMemberTable.js#L238) (copy and pasted moving around files), but the `ApplyToHousehold` component has the paths when it passes it in and then it would be path when coming from `HeadOfHousehold` and `Dependent` so best to keep it consistent.

It's also needed for Becky's PR so I would still keep it

Reply:
Thanks